compiler/luci-interpreter/src/kernels/TransposeConv.test.cpp

```diff
@@ -50,8 +50,8 @@ void Check(std::initializer_list<int32_t> output_shape_shape,
   params.stride_height = stride_height;
   params.stride_width = stride_width;
 
-  TransposeConv kernel(&output_shape_tensor, &weight_tensor, &input_data_tensor, &output_tensor,
-                       params);
+  TransposeConv kernel(&output_shape_tensor, &weight_tensor, &input_data_tensor, nullptr,
```

Comment:
It would be nice to test non-zero bias.

Reply:
okay, I will add a new test case for non-zero bias test case. after that, I will make a notification.